diagnostics/model_test/mnist_cnn.json

```diff
@@ -0,0 +1,11 @@
+{
+    "model_name": "mnist_cnn",
+    "frozen_graph_location": "/nfs/site/home/mingshan/frozen_graphs/frozen_mnist.pb",
+    "input_tensor_name": ["import/Placeholder:0"],
```

Comment:
the .pb is in an internal website. This is fine I suppose, since people are supposed to use their own .pb for debugging. But if we wanted someone to try this script, it would not run (unless we checkin the mnist .pb in our repo).

Reply:
No this is fine. In fact when we use this framework for ci, we could checkin the sons for other networks like resnet. They may not be runnable by external folks, but could still serve as an example.